Assets/Scripts/UIManager.cs

```diff
@@ -0,0 +1,321 @@
+
+                        case "Capsule":
+                            summonImage1.sprite = zombieImage;
+                            break;
+
+                    }
+                    break;
+                case 1:
+                    switch (summon)
+                    {
+                        case "Cube":
+                            summonImage2.sprite = ghostImage;
+                            break;
+
+                        case "Sphere":
+                            summonImage2.sprite = skeletonImage;
+                            break;
+
+                        case "Capsule":
+                            summonImage2.sprite = zombieImage;
+                            break;
+
+                    }
+                    break;
+                case 2:
+                    switch (summon)
+                    {
+                        case "Cube":
+                            summonImage3.sprite = ghostImage;
+                            break;
+
+                        case "Sphere":
+                            summonImage3.sprite = skeletonImage;
+                            break;
+
+                        case "Capsule":
+                            summonImage3.sprite = zombieImage;
+                            break;
+
+                    }
+                    break;
+                case 3:
+                    switch (summon)
+                    {
+                        case "Cube":
+                            summonImage4.sprite = ghostImage;
+                            break;
+
+                        case "Sphere":
+                            summonImage4.sprite = skeletonImage;
+                            break;
+
+                        case "Capsule":
+                            summonImage4.sprite = zombieImage;
+                            break;
+
+                    }
+                    break;
+                case 4:
+                    switch (summon)
+                    {
+                        case "Cube":
+                            summonImage5.sprite = ghostImage;
+                            break;
+
+                        case "Sphere":
+                            summonImage5.sprite = skeletonImage;
+                            break;
+
+                        case "Capsule":
+                            summonImage5.sprite = zombieImage;
+                            break;
+
+                    }
+                    break;
+            
+
+            /*switch (summon)
+            {
+                case "Ghost":
+                    summonImage.sprite = ghostImage;
+                    break;
+
+                case "Skeleton":
+                    summonImage.sprite = skeletonImage;
+                    break;
+
+                case "Zombie":
+                    summonImage.sprite = zombieImage;
+                    break;
+
+            }*/
+
+        }
+    }
+
+    public void UpdateSummonName(string name, int summonCount)
+    {
```

Comment:
Same as the UpdateSummonImage method, but instead of the image component pass the text component

Reply:
Done